### PR TITLE
Fixed MD issues and updated Gemini faucet instructions

### PR DESCRIPTION
**Pull request by alfetopito**
As title states.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr225--dexreact.review.gnosisdev.com

**Comment by alfetopito** [comment]
> I see most of the lines changed because of removing `*`. Made it very hard to see the differences.

Sorry. Should have split into two commits: 1 for linter changes and 1 for actual changes.

Still, you can diff the MD formatted version as well. Nice feature!

File: src/api/tokenList/Faucet_info.md [hunk 1]
```diff
@@ -1,51 +1,61 @@
...
-* **Rinkeby faucet**: [https://faucet.rinkeby.io/](https://faucet.rinkeby.io)
 
-## Tether:
-* [https://tether.to](https://tether.to)
-* Fiat enabled collateralized stable coin, that is backed by the most popular fiat currency, USD (US Dollar) in a 1:1 ratio
-* **Rinkeby faucet**: contract created with Team's shared account 0xf85D1a0E1b71e72013Db51139f285C6d5356B712.
-Use it to mint new tokens
+- [https://weth.io](https://weth.io)
+- Wrapper of Ether to make it ERC-20 compliant
+- **Rinkeby faucet**: [https://faucet.rinkeby.io/](https://faucet.rinkeby.io)
```
Comment (Velenir) [inline src/api/tokenList/Faucet_info.md]: You don't need `[url](url)` when it's the same url. https://faucet.rinkeby.io/ would have been enough

Just an fyi, no need to change

Comment (bh2smith) [reply src/api/tokenList/Faucet_info.md]: Nice to know!

Comment (anxolin) [reply src/api/tokenList/Faucet_info.md]: True, although not all MD implementation support that. I think Github does, so I agree it should be enough in this case.

Comment (alfetopito) [reply src/api/tokenList/Faucet_info.md]: I only `fixed` it because VScode MD linter complained about it.

**Pull request merged**